src/test/java/org/datadog/jenkins/plugins/datadog/clients/DatadogClientTest.java

```diff
@@ -101,6 +108,34 @@ public void testDogstatsDClientGetInstanceEnableValidations() {
         Assert.assertEquals(client, null);
     }
 
```

Comment:
Would it be possible to test `fetchAgentSupportedEndpoints`? Such as confirm if there's no agent available, it will return an empty list